model/src/test/kotlin/PackageCurationTest.kt

```diff
@@ -34,22 +34,8 @@ import org.ossreviewtoolkit.utils.spdx.toSpdx
 class PackageCurationTest : WordSpec({
     "Applying a single curation" should {
         "overwrite the correct values" {
-            val pkg = Package(
-                id = Identifier(
-                    type = "Maven",
-                    namespace = "org.hamcrest",
-                    name = "hamcrest-core",
-                    version = "1.3"
-                ),
-                authors = emptySet(),
-                declaredLicenses = setOf("license a", "license b"),
-                description = "",
-                homepageUrl = "",
-                binaryArtifact = RemoteArtifact.EMPTY,
-                sourceArtifact = RemoteArtifact.EMPTY,
-                vcs = VcsInfo.EMPTY,
-                isMetadataOnly = false,
-                isModified = false
+            val pkg = Package.EMPTY.copy(
```

Comment:
The commit message sounds to me as if the intention was to generate exactly the same code, but simpler. However, the generate code is changed as e.g. the identifier is not set to a "Maven" one anymore.

If this is really intended, it would be great to make that more clear by saying something like "simply code by using empty values where non-empty values are not required for the test".